### ISSUE TITLE
[Feature Request] Support for PHP 8.1

### ISSUE THREAD

**Issue by die2mrw007** [litespeedtech/cyberpanel#780]
lsphp has released PHP 8.1 with all dependencies. I manually tested it and its all working fine.

Request to add lsphp81 to cyberpanel install.

I tried this on my Almalinux setup : https://rpms.litespeedtech.com/centos/8/x86_64/RPMS/

**Comment by usmannasir** [comment]
I will take a look into this.

**Comment by usmannasir** [comment]
Added.

**Comment by dfag86** [comment]
![IMG_20211227_212716.jpg](https://user-images.githubusercontent.com/92047835/147505144-c9f44436-9954-4aaa-9b18-f6da215091f7.jpg)

**Comment by dfag86** [comment]
![IMG_20211227_212706.jpg](https://user-images.githubusercontent.com/92047835/147505154-558873b6-91e0-47de-b677-eca5460ab2e9.jpg)

**Comment by ddegner** [comment]
I have php 8.1 installed on my litespeed server and ran the cyberpanel upgrade script, but still don't see 8.1 on the Edit PHP Config page.

I need to upgrade because lsphp80-imagick is still giving "PHP_IMAGICK_VERSION "@PACKAGE_VERSION@"" which means that it is installed but wordpress won't use it.

**Comment by nickchomey** [comment]
I get these same errors. I ran apt install lsphp81 and now I can see the Basic and Advanced tabs for php 8.1 config. But I get this error when I go to install extensions: `Cannot fetch details. Error message: PHP matching query does not exist.`

It looks like lsphp81 is missing many extensions anyway, so I'm not going to worry about it. But something within CyberPanel seems to be broken with regards to 8.1

`apt-cache search lsphp*`

lsphp74 - server-side, HTML-embedded scripting language (LSAPI binary)
lsphp74-apcu - APC User Cache for PHP
lsphp74-common - Common files for packages built from the PHP source
lsphp74-curl - CURL module for PHP
lsphp74-dbg - Debug symbols for LSPHP70
lsphp74-dev - Files for PHP70 module development
lsphp74-igbinary - igbinary PHP serializer
lsphp74-imagick - Provides a wrapper to the ImageMagick library
lsphp74-imap - IMAP module for PHP
lsphp74-intl - Internationalisation module for PHP
lsphp74-ioncube - PHP extension for interfacing with ioncube
lsphp74-json - JSON module for PHP
lsphp74-ldap - LDAP module for PHP
lsphp74-memcached - memcached extension module for PHP, uses libmemcached
lsphp74-modules-source - PHP 70 modules source package
lsphp74-msgpack - PHP extension for interfacing with MessagePack
lsphp74-mysql - MySQL module for PHP
lsphp74-opcache - Zend OpCache module for PHP
lsphp74-pear - Provides PHP PEAR package
lsphp74-pgsql - PostgreSQL module for PHP
lsphp74-pspell - pspell module for PHP
lsphp74-redis - PHP extension for interfacing with Redis
lsphp74-snmp - SNMP module for PHP
lsphp74-sqlite3 - SQLite3 module for PHP
lsphp74-sybase - Sybase module for PHP
lsphp74-tidy - tidy module for PHP
lsphp80 - server-side, HTML-embedded scripting language (LSAPI binary)
lsphp80-apcu - APC User Cache for PHP
lsphp80-common - Common files for packages built from the PHP source
lsphp80-curl - CURL module for PHP
lsphp80-dbg - Debug symbols for LSPHP70
lsphp80-dev - Files for PHP70 module development
lsphp80-igbinary - igbinary PHP serializer
lsphp80-imagick - Provides a wrapper to the ImageMagick library
lsphp80-imap - IMAP module for PHP
lsphp80-intl - Internationalisation module for PHP
lsphp80-ldap - LDAP module for PHP
lsphp80-memcached - memcached extension module for PHP, uses libmemcached
lsphp80-modules-source - PHP 70 modules source package
lsphp80-msgpack - PHP extension for interfacing with MessagePack
lsphp80-mysql - MySQL module for PHP
lsphp80-opcache - Zend OpCache module for PHP
lsphp80-pear - Provides PHP PEAR package
lsphp80-pgsql - PostgreSQL module for PHP
lsphp80-pspell - pspell module for PHP
lsphp80-redis - PHP extension for interfacing with Redis
lsphp80-snmp - SNMP module for PHP
lsphp80-sqlite3 - SQLite3 module for PHP
lsphp80-sybase - Sybase module for PHP
lsphp80-tidy - tidy module for PHP
lsphp81 - server-side, HTML-embedded scripting language (LSAPI binary)
lsphp81-common - Common files for packages built from the PHP source
lsphp81-curl - CURL module for PHP
lsphp81-dbg - Debug symbols for LSPHP70
lsphp81-dev - Files for PHP70 module development
lsphp81-imap - IMAP module for PHP
lsphp81-intl - Internationalisation module for PHP
lsphp81-ldap - LDAP module for PHP
lsphp81-modules-source - PHP 70 modules source package
lsphp81-mysql - MySQL module for PHP
lsphp81-opcache - Zend OpCache module for PHP
lsphp81-pgsql - PostgreSQL module for PHP
lsphp81-pspell - pspell module for PHP
lsphp81-snmp - SNMP module for PHP
lsphp81-sqlite3 - SQLite3 module for PHP
lsphp81-sybase - Sybase module for PHP
lsphp81-tidy - tidy module for PHP

**Comment by nickchomey** [comment]
I'm still getting similar errors regarding lsphp81. And there are still some modules missing for 80, such as redis. I believe these have all been released by now.

**Comment by nickchomey** [comment]
I just upgraded Cyberpanel and lsphp80 is missing some extensions. Namely Redis. 
![image](https://user-images.githubusercontent.com/88559987/168489383-fd2f4cbb-d503-42ab-bea7-6157e688b6ff.png)

php 8.1. still give these errors 
![image](https://user-images.githubusercontent.com/88559987/168489415-3ae1ee2a-e55c-4557-9728-46edd7e23d62.png)

![image](https://user-images.githubusercontent.com/88559987/168489423-4cdfecd0-a150-4d8a-925b-d8486a29c022.png)

But almost all of the extensions are actually installed on my server, seen with `apt-cache search lsphp*`

```
lsphp80 - server-side, HTML-embedded scripting language (LSAPI binary)
lsphp80-apcu - APC User Cache for PHP
lsphp80-common - Common files for packages built from the PHP source
lsphp80-curl - CURL module for PHP
lsphp80-dbg - Debug symbols for LSPHP70
lsphp80-dev - Files for PHP70 module development
lsphp80-igbinary - igbinary PHP serializer
lsphp80-imagick - Provides a wrapper to the ImageMagick library
lsphp80-imap - IMAP module for PHP
lsphp80-intl - Internationalisation module for PHP
lsphp80-ldap - LDAP module for PHP
lsphp80-memcached - memcached extension module for PHP, uses libmemcached
lsphp80-modules-source - PHP 70 modules source package
lsphp80-msgpack - PHP extension for interfacing with MessagePack
lsphp80-mysql - MySQL module for PHP
lsphp80-opcache - Zend OpCache module for PHP
lsphp80-pear - Provides PHP PEAR package
lsphp80-pgsql - PostgreSQL module for PHP
lsphp80-pspell - pspell module for PHP
lsphp80-redis - PHP extension for interfacing with Redis
lsphp80-snmp - SNMP module for PHP
lsphp80-sqlite3 - SQLite3 module for PHP
lsphp80-sybase - Sybase module for PHP
lsphp80-tidy - tidy module for PHP
lsphp81 - server-side, HTML-embedded scripting language (LSAPI binary)
lsphp81-apcu - APC User Cache for PHP
lsphp81-common - Common files for packages built from the PHP source
lsphp81-curl - CURL module for PHP
lsphp81-dbg - Debug symbols for LSPHP70
lsphp81-dev - Files for PHP70 module development
lsphp81-igbinary - igbinary PHP serializer
lsphp81-imagick - Provides a wrapper to the ImageMagick library
lsphp81-imap - IMAP module for PHP
lsphp81-intl - Internationalisation module for PHP
lsphp81-ldap - LDAP module for PHP
lsphp81-memcached - memcached extension module for PHP, uses libmemcached
lsphp81-modules-source - PHP 70 modules source package
lsphp81-msgpack - PHP extension for interfacing with MessagePack
lsphp81-mysql - MySQL module for PHP
lsphp81-opcache - Zend OpCache module for PHP
lsphp81-pgsql - PostgreSQL module for PHP
lsphp81-pspell - pspell module for PHP
lsphp81-redis - PHP extension for interfacing with Redis
lsphp81-snmp - SNMP module for PHP
lsphp81-sqlite3 - SQLite3 module for PHP
lsphp81-sybase - Sybase module for PHP
lsphp81-tidy - tidy module for PHP

```

Both 80  and 81 are missing `ioncube` and `json`, and 81 is missing `pear` as well.

And the list in CyberPanel for 80 is compeltely different from what I found and listed above.

**Comment by usmannasir** [comment]
This is fixed in `v2.3.2-dev`
